src/components/Materialize/Dropdown.vue

```diff
@@ -55,7 +55,7 @@
 
         let parsed = this.id + this._uid
 
-        return parsed.replace(/[!"#$%&'()*_+,./:;<=>?@[\]^`{|}~ ]/g, '\\$&')
+        return parsed.replace(/[!"#$%&'()*+,./:;<=>?@[\]^`{|}~ ]/g, '\\$&')
```

Comment:
see https://github.com/kuzzleio/kuzzle-backoffice/pull/361

Reply:
Maybe we can use this fix for the moment because we need to dig deep in materialize to propose a perfect fix

Reply:
I fail to see how this half-hearted fix can help in any way. And I also don't see how this PR fixes #350.

Reply:
Ok, we should keep this issue open